src/services/ChallengeService.js

```diff
@@ -294,7 +294,8 @@ searchChallenges.schema = {
     updatedBy: Joi.string(),
     memberId: Joi.number().integer().positive(),
     sortBy: Joi.string().valid(_.values(constants.validChallengeParams)),
-    sortOrder: Joi.string().valid(['asc', 'desc'])
+    sortOrder: Joi.string().valid(['asc', 'desc']),
+    groupIds: Joi.array().items(Joi.optionalId()).unique().min(1)
```

Comment:
Does this JOI mean that I have to pass at least one groupId?, or only if groupIds is set at all ( because it's not required )

Reply:
@rootelement - This Joi rule mean that if `groupIds` is set it must have at least one element, but `groupIds` is optional.

Reply:
Perfect. Just confirming. Will merge in and test. Thanks!

Reply:
Ok. Thank you too.